### PR TITLE
CHAOS-1635: Adds chaos prerequisites, moves topic

### PR DESCRIPTION
**Pull request by StrawberryPad**
[**Link to Preview**](https://649ed545b7cda00e77697c31--harness-developer.netlify.app/docs/chaos-engineering/configure-chaos-experiments/prerequisites)
Note this link is good, but I just corrected a typo (environmments)

### CODE REVIEW

**Review by uditgaurav — DISMISSED** [review]

File: docs/chaos-engineering/configure-chaos-experiments/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+The following are the chaos infrastructure Execution Plane components and required resources . These components are installed in your target cluster, allowing the chaos infrastructure to run experiments there. Chaos infrastructure runs within your target environment to aid CE in accessing the target resources and injecting chaos at a cloud-native scale.
+
+| Deployment | Container | CPU<br />required | Memory<br />required | Image |
+|------------|-----------|-------------------|----------------------|-------|
+| chaos-operator-ce  | chaos-operator-ce     | 125m | 300M | chaosnative/chaos-operator          |
+| chaos-exporter     | chaos-exporter        | 125m | 300M | chaosnative/chaos-exporter          |
+| subscriber         | subscriber            | 125m | 300M | chaosnative/harness-chaos-subscriber|
+| workflow-controller| workflow-controller   | 125m | 300M | chaosnative/workflow-controller     |
+
```
Comment (uditgaurav) [inline docs/chaos-engineering/configure-chaos-experiments/prerequisites.md:18]: Looks Good. I hope the other prerequisites are placed somewhere else like k8s cluster, kubectl.

File: docs/chaos-engineering/configure-chaos-experiments/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: Prerequisites
+sidebar_position: 1
+---
+
+This topic describes the requirements and resources you need to run chaos experiments on your target environments.
+
+## Chaos infrastructure requirements
+
+The following are the chaos infrastructure Execution Plane components and required resources . These components are installed in your target cluster, allowing the chaos infrastructure to run experiments there. Chaos infrastructure runs within your target environment to aid CE in accessing the target resources and injecting chaos at a cloud-native scale.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/configure-chaos-experiments/prerequisites.md:10]:
```suggestion
The following are the chaos infrastructure Execution Plane components and required resources. These components are installed in your target cluster, allowing the chaos infrastructure to run experiments there. Chaos infrastructure runs within your target environment to aid CE in accessing the target resources and injecting chaos at a cloud-native scale.
```

File: docs/chaos-engineering/configure-chaos-experiments/prerequisites.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+	1. In **New User(s)**, add the email(s) of the user(s) you want to add. 
+	1. In **User Groups**, select any of the existing user groups to which the user should be added. 
+	1. Add the roles for the user(s) to enforce their access permissions. 
+
+	![Add New User](./static/add-new-user.png)
+
+1. Select **Apply**.
+
+This will send an email invite to the new user to add themselves to the project.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/configure-chaos-experiments/prerequisites.md:43]:
```suggestion
This sends an email invite to the new user to add themselves to the project.
```

**Review by brian-f-harness — DISMISSED** [review]
lgtm

**Pull request merged**